app/ui/browser-blueprint/views/overview/overview-cards.jsx

```diff
@@ -32,7 +33,12 @@ const OverviewCards = function(props) {
       {props.pages.map((page, pageIndex) => (
         <OverviewCard key={`page-${page.id}`}
           page={page}
-          pageIndex={pageIndex}
+          width={UIConstants.CARD_WIDTH}
```

Comment:
If  all of the front ends should have the same value (at least in the short term), I'd be tempted to say we should declare the constants directly in `overview-card.js`. I think that would make more sense until we have themes that require different values.